README.md

```diff
@@ -192,21 +193,21 @@ The following table provides a few of the available variables:
 | MESSAGE_LOG_EXI   | `False`                       | Whether or not to log the EXI Bytestream messages (only works if log level is set to DEBUG)
 
 
-## Licence
+## License
```

Comment:
I think this is fine -- American English is often the standard in a technical context.